bot/exts/evergreen/trivia_quiz.py

```diff
@@ -20,46 +21,22 @@
 STANDARD_VARIATION_TOLERANCE = 88
 DYNAMICALLY_GEN_VARIATION_TOLERANCE = 97
 
+MAX_ERROR_FETCH_TRIES = 3
+
 WRONG_ANS_RESPONSE = [
     "No one answered correctly!",
     "Better luck next time...",
 ]
 
-N_PREFIX_STARTS_AT = 5
-N_PREFIXES = [
-    "penta", "hexa", "hepta", "octa", "nona",
-    "deca", "hendeca", "dodeca", "trideca", "tetradeca",
-]
-
-PLANETS = [
-    ("1st", "Mercury"),
-    ("2nd", "Venus"),
-    ("3rd", "Earth"),
-    ("4th", "Mars"),
-    ("5th", "Jupiter"),
-    ("6th", "Saturn"),
-    ("7th", "Uranus"),
-    ("8th", "Neptune"),
-]
+RULES = (
+    "No cheating and have fun!",
+    "Points for each question reduces by 25 after 10s or after a hint. Total time is 30s per question"
+)
 
-TAXONOMIC_HIERARCHY = [
-    "species", "genus", "family", "order",
-    "class", "phylum", "kingdom", "domain",
-]
-
-UNITS_TO_BASE_UNITS = {
-    "hertz": ("(unit of frequency)", "s^-1"),
-    "newton": ("(unit of force)", "m*kg*s^-2"),
-    "pascal": ("(unit of pressure & stress)", "m^-1*kg*s^-2"),
-    "joule": ("(unit of energy & quantity of heat)", "m^2*kg*s^-2"),
-    "watt": ("(unit of power)", "m^2*kg*s^-3"),
-    "coulomb": ("(unit of electric charge & quantity of electricity)", "s*A"),
-    "volt": ("(unit of voltage & electromotive force)", "m^2*kg*s^-3*A^-1"),
-    "farad": ("(unit of capacitance)", "m^-2*kg^-1*s^4*A^2"),
-    "ohm": ("(unit of electric resistance)", "m^2*kg*s^-3*A^-2"),
-    "weber": ("(unit of magnetic flux)", "m^2*kg*s^-2*A^-1"),
-    "tesla": ("(unit of magnetic flux density)", "kg*s^-2*A^-1"),
-}
+RANDOM_WIKI_URL = "https://en.wikipedia.org/api/rest_v1/page/random/summary"
+TRIVIA_QUIZ_ICON = (
+    "https://raw.githubusercontent.com/python-discord/branding/main/icons/trivia_quiz/trivia-quiz-dist.png"
+)
 
 
 @dataclass(frozen=True)
```

Comment:
1. Why is `answer` not a set/list of allowed answers?
2. I would remove words like "dataclass" and repeating the class name, which are already clear without the docstring.

Reply:
Doing the first change falls out of the scope of the PR, and requires a bit amount of refactoring so I am not sure if I should include that here.